isis/CMakeLists.txt

```diff
@@ -40,8 +40,8 @@ set(PACKAGE_VERSION    ${VERSION})
 set(PACKAGE_STRING     "${PACKAGE_NAME} ${VERSION}")
 
 # Other release information
-set(VERSION_DATE              "2018-10-26")
-set(RELEASE_STAGE             "beta") # (alpha, beta, stable)
+set(VERSION_DATE              "2019-05-29")
+set(RELEASE_STAGE             "stable") # (alpha, beta, stable)
```

Comment:
Hmm... I thought that this file had already been updated recently...? Was this a change you made specifically for this PR?

Reply:
No, that was a mistake

Reply:
@SgStapleton Fixed.